Missing include for `std::for_each`

When building on Ubuntu 16.04 with CMake and default GCC 5.4.0, build fails with the following error message:

    config.cpp:91:4: error: ‘for_each’ is not a member of ‘std’

This is due to a missing include statement for `<algorithm>`

--- a/src/config.cpp
+++ b/src/config.cpp
@@ -20,6 +20,7 @@
  * ******************************************************************/
 
 #include "config.h"
+#include <algorithm>
 #include <fstream>
 #include <limits>
 #include <cstring>

